include/RAJA/pattern/kernel/Region.hpp

```diff
@@ -48,34 +46,21 @@ namespace internal
 
 //Note: RAJA region's lambda must capture by reference otherwise
 //internal function calls are undefined.
-template<typename RegionPolicy, typename... EnclosedStmts>
-struct StatementExecutor<statement::Region<RegionPolicy, EnclosedStmts...> > {
+template<typename RegionPolicy, typename... EnclosedStmts, typename Types>
```

Comment:
I've noticed that we have EnclosedStmts followed by Types in some template parameter lists and sometimes Types followed by EnclosedStmts.  Is there a clear reason for this?

Reply:
Where does Types come first?   I tried to make that consistent.

In the end it doesn't matter, since it should drop into the specialization the same way

Reply:
I don't recall exactly where. But in looking over this PR, there are many instances of each ordering. I agree that it doesn't matter. I hope it doesn't get confusing. Having a variadic template parameter in the middle of a template arg list makes my head hurt.